src/steps/resource-manager/monitor/converters.ts

```diff
@@ -0,0 +1,33 @@
+import { AzureWebLinker } from '../../../azure';
+import {
+  createIntegrationEntity,
+  Entity,
+} from '@jupiterone/integration-sdk-core';
+import { MonitorEntities } from './constants';
+import { LogProfileResource } from '@azure/arm-monitor/esm/models';
+
+export function createMonitorLogProfileEntity(
+  webLinker: AzureWebLinker,
+  data: LogProfileResource,
+): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: data,
+      assign: {
+        _key: data.id,
+        _type: MonitorEntities.MONITOR_LOG_PROFILE._type,
+        _class: MonitorEntities.MONITOR_LOG_PROFILE._class,
+        id: data.id,
+        webLink: webLinker.portalResourceUrl(data.id),
+        name: data.name,
+        displayName: data.name,
+        storageAccountId: data.storageAccountId,
+        serviceBusRuleId: data.serviceBusRuleId,
+        locations: data.locations,
+        categories: data.categories,
+        'retentionPolicy.enabled': data.retentionPolicy.enabled,
+        'retentionPolicy.days': data.retentionPolicy.days,
```

Comment:
Is `data.retentionPolicy` nullable? If so I think you'll have to add optional chaining here.

```suggestion
        'retentionPolicy.enabled': data.retentionPolicy?.enabled,
        'retentionPolicy.days': data.retentionPolicy?.days,
```

Reply:
According to the typings, it is not nullable.
<img width="1069" alt="Screen Shot 2020-11-10 at 10 05 23 AM" src="https://user-images.githubusercontent.com/6044786/98706653-52da9a00-233c-11eb-94e1-40601b1858e8.png">